app/views/layout.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <!-- 
```

Comment:
This is fine, but be careful about opening and just changing whitespace on files. Try not to commit files that you didn't intend to change. @DanielJPark